config/all-in-one/templates/resource-locker.yaml

```diff
@@ -12,7 +12,7 @@
   {{- end }}
 {{- end }}
 
-{{- if eq .Values.cp4waiops.profile "x-small" }}
+{{- if or (eq .Values.cp4waiops.profile "x-small")  (eq .Values.cp4waiops.profile "x-small-idle")  (eq .Values.cp4waiops.profile "x-small-custom") }}
```

Comment:
Suggest to use [hasprefix or hassuffix](https://helm.sh/docs/chart_template_guide/function_list/#hasprefix-and-hassuffix), e.g.: everything started w/ `x-small` will fall into this case.

Reply:
Agree to use prefix.